src/waveform/waveformwidgetfactory.cpp

```diff
@@ -227,6 +228,9 @@ bool WaveformWidgetFactory::setConfig(UserSettingsPointer config) {
     bool showBeatGrid = m_config->getValue(ConfigKey("[Waveform]", "beatGridLinesCheckBox"), m_beatGridEnabled);
     setDisplayBeatGrid(showBeatGrid);
 
+    bool showBeatGridAlpha = m_config->getValue(ConfigKey("[Waveform]", "dimModeCheckBox"), m_beatGridAlpha);
```

Comment:
The parameter should called what it effects, not how it is set. 
"semiTransparentBeatGrid true / false" or "beatGridAlpha 0.3 / 0.9"
In the later case you can still use a checkbox to set one of these values. If we later deside to make the alpha value a skin parameter, the first one works better. The later allows to control the alpha by a slider.